_pages/meetings.md

```diff
@@ -43,13 +43,7 @@ _pages/meetings/april2020.md %}) **will be held remotely** on Zoom (see details
 
 #### AGU Sessions
 
-AGU 2020 Call for Abstracts is now open! PyHC will be convening the following session:
+* [SH010 - Python Application Development for Solar and Space Physics (Poster session)](https://agu.confex.com/agu/fm20/meetingapp.cgi/Session/102187), Tuesday, 8 December 2020  0:400 - 20:59 PST)
```

Comment:
There’s a closing paren ) at the end here that’s out of place. Looks good otherwise!

Reply:
Fixed that, thanks!